src/medgen2obo.pl

```diff
@@ -133,14 +135,27 @@
                     $tag = 'is_a:';
                 }
                 if ($rel eq 'mapped_to') {
-                    $tag = 'equivalent_to:';
```

Comment:
@matentzn Having trouble replacing the `owl:equivalentClass` with `skos:exactMatch` in a clean/proper way. I'd like to edit the original Perl that creates the OBO to do this if possible.

Reached on on [Slack](https://obo-communitygroup.slack.com/archives/C01BDKWDS91/p1691451466478739).

Reply:
It's best if you follow the conventions of the other ingests - it looks like GARD uses skos natively so that would be the best choice.

Reply:
@cmungall So, that is what we are trying to do. GARD does not use any mapping predicates. They have 2 columns, 1 with the GARD ID, and another with e.g. OMIM ID, and we interpret that an exact match. 
For MedGen, it looks like the same situation, e.g. from `MedGenIDMappings.txt`:
```csv
#CUI|pref_name|source_id|source|
C0000727|Acute abdomen|1674|MedGen|
```

So we want to use the same convention. But my problem is with implementation.
I am manipulating the existing Perl pipeline to create OBO, such that it will be converted into OWL.
What do I need to output into OBO, such that when converted to OWL, I get `skos:exactMatch`?

Reply:
I believe I've figured out how to do this. In OBO, this:
`equivalent_to: <CURIE>`
Should be changed to this:
`property_value: exactMatch <CURIE>`